@jameslnewell Halloo! I tried firing this up locally, but ran into issues 😅 

```
ERROR in ../components/packages/typography/src/body/index.tsx
Module not found: Error: Can't resolve '@salad-ui/box' in '/Users/Q/Sites/@salad-ui/components/packages/typography/src/body'
 @ ../components/packages/typography/src/body/index.tsx 15:0-36 35:23-26
 @ ../components/packages/typography/src/index.ts
 @ ./app/manager/index.tsx

ERROR in ../components/packages/typography/src/caption/index.tsx
Module not found: Error: Can't resolve '@salad-ui/box' in '/Users/Q/Sites/@salad-ui/components/packages/typography/src/caption'
 @ ../components/packages/typography/src/caption/index.tsx 15:0-36 20:26-29
 @ ../components/packages/typography/src/index.ts
 @ ./app/manager/index.tsx

ERROR in ../components/packages/typography/src/subtitle/index.tsx
Module not found: Error: Can't resolve '@salad-ui/box' in '/Users/Q/Sites/@salad-ui/components/packages/typography/src/subtitle'
 @ ../components/packages/typography/src/subtitle/index.tsx 15:0-36 36:27-30
 @ ../components/packages/typography/src/index.ts
 @ ./app/manager/index.tsx

ERROR in ../components/packages/typography/src/title/index.tsx
Module not found: Error: Can't resolve '@salad-ui/box' in '/Users/Q/Sites/@salad-ui/components/packages/typography/src/title'
 @ ../components/packages/typography/src/title/index.tsx 15:0-36 39:24-27
 @ ../components/packages/typography/src/index.ts
 @ ./app/manager/index.tsx
```

I have docker running. And the `@salad-ui/components` repo on my computer.
Ran `yarn` for both projects

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -1,7 +1,6 @@
 version: '3.3'
 
 services:
-
   mysql:
     image: mysql:5.7
     volumes:
@@ -20,13 +19,14 @@ services:
     volumes:
       - ./plugin/:/var/www/html/wp-content/plugins/wp-media-center
     ports:
-      - "8000:80"
+      - '8000:80'
     restart: always
     environment:
       WORDPRESS_DB_HOST: mysql:3306
       WORDPRESS_DB_USER: wordpress
       WORDPRESS_DB_PASSWORD: wordpress
       WORDPRESS_DB_NAME: wordpress
+      WP_MEDIA_CENTER_ENVIRONMENT: development
 
 volumes:
   mysql_data: {}

--- a/plugin/wp-media-center.php
+++ b/plugin/wp-media-center.php
@@ -7,4 +7,23 @@
  * License URI:       https://www.gnu.org/licenses/gpl-2.0.html
  */
 
-echo 'Hello World';
+
+$environment = isset($_ENV['WP_MEDIA_CENTER_ENVIRONMENT']) ? $_ENV['WP_MEDIA_CENTER_ENVIRONMENT'] : 'production';
+$isDevelopment = $environment === 'development' ? true : false;
+
+function render_manager_page() {
+  echo('<div id="wp-media-center__manager-app"></div>');
+}
+
+function add_manager_scripts() {
+  global $isDevelopment;
+  $url = $isDevelopment ? 'http://localhost:8081/manager.js' : plugin_dir_url(__FILE__).'static/manager.js';
+  wp_enqueue_script('wp-media-center', $url, array(), null, true); 
+}
+
+function add_manager_menu() {
+  $managerHookSuffix = add_menu_page('Media Center', 'Media Center', 'manage_options', 'media-center', 'render_manager_page', 'dashicons-admin-media', 10);
+  add_action('admin_print_scripts-'.$managerHookSuffix, 'add_manager_scripts');
+}
+
+add_action('admin_menu', 'add_manager_menu');

--- a/webpack.config.js
+++ b/webpack.config.js
@@ -9,7 +9,7 @@ module.exports = {
     manager: './app/manager/index.tsx',
   },
   output: {
-    path: `${__dirname}/plugin/static`,
+    path: path.resolve(__dirname, `plugin/static`),
   },
   resolve: {
     extensions: ['.ts', '.tsx', '.js'],
@@ -39,12 +39,12 @@ module.exports = {
   plugins: [
     new HtmlWebpackPlugin({
       template: './app/chooser/index.html',
-      filename: './chooser/index.html',
+      filename: './chooser.html',
       excludeChunks: ['manager'],
     }),
     new HtmlWebpackPlugin({
       template: './app/manager/index.html',
-      filename: './manager/index.html',
+      filename: './manager.html',
       excludeChunks: ['chooser'],
     }),
     new ForkTsCheckerWebpackPlugin(),

